### ISSUE TITLE
Discussion: (New report) Company overview

### ISSUE THREAD

**Issue by tarjeieo** [Puzzlepart/did#241]
### TLDR
Summary view of all hours in the organization, presented per customer.

### Description

In the classic Did we had an overview over all customers per week&month, see https://did.puzzlepart.com/Customers.aspx/Period/2020-Q2-04. The "aggregate by customer" way of presenting this is very useful for those who are creating invoices.

AFAIK this is missing in Did365? The summary view of https://did365.puzzlepart.com/timesheet is similar, but is only for the current user.

What's the complexity for implementing this for all reported hours in the organization? 

My 2 cents; Think this would be useful, but could also be considered to be a PowerBI report? Drilling down into the hours from the same view would also be super-useful.

![image](https://user-images.githubusercontent.com/1837390/79451001-159c6c80-7fe6-11ea-9b60-1d190907b48a.png)
